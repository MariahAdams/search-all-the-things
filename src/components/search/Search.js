import React, { Component } from 'react';
import styles from './Search.css';

class Search extends Component {

  render() {

    return (
      <div className={styles.search}>
        <h3>Search Component</h3>
      </div>
    )
  }
}

export default Search;