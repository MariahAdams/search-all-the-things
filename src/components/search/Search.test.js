import React from 'react';
import { shallow, mount } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Search from './Search';

describe('Search', () => {

  it('testing...', () => {

  });
});