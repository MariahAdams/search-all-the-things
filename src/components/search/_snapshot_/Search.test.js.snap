// Jest Snapshot v1, docs: https://goo.gl/fbAQLP

exports[`Search renders as designed 1`] = `
<form 
  className="search-form" 
  onSubmit={[Function]}
>
  <label>
    Search For:
    <input 
      value={search} 
      onChange={this.handleChange}
    />
  </label>
  <button>
    Search
  </button>
</form>
`;